lncrawl/core/crawler.py

```diff
@@ -106,7 +106,7 @@ def index_of_chapter(self, url: str) -> int:
         """Return the index of chapter by given url or 0"""
         url = self.absolute_url(url)
         for chapter in self.chapters:
-            if chapter.url == url:
+            if chapter.url.rstrip("/") == url.rstrip("/"):
```

Comment:
```if chapter.url.rstrip("/") == url:```

bcuz ```url = self.absolute_url(url)```

absolute_url returns url without right "/"

Reply:
Changed it